packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx

```diff
@@ -170,6 +168,7 @@ function CodeMirror(props: NoteBodyEditorProps, ref: any) {
 						textItalic: () => wrapSelectionWithStrings('*', '*', _('emphasised text')),
 						textLink: async () => {
 							const url = await dialogs.prompt(_('Insert Hyperlink'));
+							editorRef.current.focus();
 							if (url) wrapSelectionWithStrings('[', `](${url})`);
```

Comment:
Any reason why focus is hard-coded here?

Reply:
This is the definition of the `textLink` command. If this command was included as an editor command then the popup box wouldn't be focused and the editor would get the focus instead. I thought it was okay for this command (and any other like it) to be a special case of focus, because they are more complex in regards to handling focus i.e. they need to move focus around, not just on the editor.